user/enterprise/troubleshooting-guide.md

```diff
@@ -176,3 +176,21 @@ $ travis console
 ```
 
 It can happen that organizations are also stuck in the syncing state. Since an organization itself does not have a `is_syncing` flag, all users that do belong to it have to be successfully synced.
+
+## GitHub validation errors and ` response_status=422 maximum_number_of_statuses` messages observed in logs
+
+### The Problem
+
+On every commit made when a build runs, a commit status is created for a given SHA. Due to GitHub’s limitations at 1,000 statuses per SHA and context within a repository, if more than 1000 status are created this leads to a validation error.
+This issue should no longer be present in GitHub Apps integrations but will be possible in Webhooks integrations.
```

Comment:
What I'm wondering about is if it makes sense to mention what could cause this error to happen?